src/MDFTextAccessibility.m

```diff
@@ -20,6 +20,13 @@
 #import "private/MDFImageCalculations.h"
 #import "private/NSArray+MDFUtils.h"
 
+#if !defined(__IPHONE_8_2) || __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_8_2
+// This constant is defined in iOS 8.2.
+// https://developer.apple.com/reference/uikit/uifontweightmedium
+const CGFloat UIFontWeightMedium = 0.230000004172325;
```

Comment:
I'm getting a conversion error here without a (CGFloat) cast.

I suspect anything after the first two digits is floating point magic AKA I suspect UIFontWeightMedium is actually 0.23.  This quietly break in the future if the (double)0.23 ends up being less than the (float)0.23.
Recommend we change this to fontWeightMedium = (CGFloat)0.23;


I'm usually hesitant to overwrite system variables as there can be namespacing issues.  (What happens if this gets bundled into a framework targeting 8.0 then linked into an App targeting 8.2+ ?)  And it looks like we can't make it static as UIFontWeightMedium isn't a compile-time constant.

It may be overkill, but consider defining the value in the isLargeForContrastRatios: method as in the following:

// UIFontWeightMedium was defined in iOS 8.2.
// https://developer.apple.com/reference/uikit/uifontweightmedium
// TODO: Refactor when are minimum iOS version is 8.2+
#if !defined(__IPHONE_8_2) || __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_8_2
const CGFloat MDFFontWeightMedium = (CGFloat)0.23;
#else
const CGFloat MDFFontWeightMedium = UIFontWeightMedium;
#endif